documents/documents.html

```diff
@@ -98,26 +99,87 @@
         </div>
       </div>
       <div class="component-main-section">
-        <h2 class="mt-2 main-intro">Introduction</h2>
         <section id="installation" class="component-container">
-            <h3>Installation Guide</h3>
-            <p class="mt-1 lh-1">To use dreamUI in your project,copy the link given below and paste it in the "head" tag of your html,above all other style sheets. </p>
-            <div class="mt-2">
-                <iframe
-                src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=0px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Clink%2520rel%253D%2522stylesheet%2522%2520href%253D%2522dreamUi%2520lnk%2522%253E%250A"
-                style="width: 494px; height: 122px; border:0; transform: scale(1); overflow:hidden;"
-                sandbox="allow-scripts allow-same-origin">
-              </iframe>
+          <h2>Installation Guide</h2>
+          <p class="mt-1 mb-1">
+            To use dreamUI in your project,copy the link given below and paste
+            it in the "head" tag of your html,above all other style sheets.
+          </p>
+          <div class="mt-2 code-snippet">
+            <iframe
+              src="https://carbon.now.sh/embed?bg=rgba%2821%2C23%2C24%2C1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=100px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Clink%2520rel%253D%2522stylesheet%2522%2520href%253D%2522dreamUi%2520lnk%2522%253E%2520%2520%2520%2520%2520%250A"
+              style="
+                width: 518px;
```

Comment:
The inline styles which you have mentioned over here you can create a utility class for it depending on the purpose.

Reply:
The inline style came with iframe from carbon UI. Do I need to change it? May be it could have some consequences